### PR TITLE
Alert

### PR DESCRIPTION
**Pull request by Rockyraven**
some button and component added

### CODE REVIEW

File: style/comp-rest.css [hunk 1]
```diff
@@ -43,4 +43,11 @@ main{
     width: 100%;
     border: none;
     min-height: 50vh;
+}
+.component-button{
+    font-size: 1.5rem;
+    margin: 1rem;
+}
+.right-button{
+    float: right;
```
Comment (varnita619) [inline style/comp-rest.css:63]: You can add a new line at the end of each file

**Review by varnita619 — APPROVED** [review]
Rest all looks good

File: style/comp-rest.css (hunk 1 above)
Comment (Rockyraven) [reply style/comp-rest.css:63]: ok I will consider

**Review by Developers-codz — APPROVED** [review]
Looks good to merge. Just do the suggested changes

**Review by sagargse — APPROVED** [review]

**Review by arjun-computer-geek — APPROVED** [review]

**Pull request merged**